.github/workflows/ci.yml

```diff
@@ -4,16 +4,18 @@ on:
   merge_group:
   pull_request:
   push:
-    branches:
-      - main
+    branches: [main]
 
 env:
+  # Force colors, which can be displayed by Github.
   CARGO_TERM_COLOR: always
+  # Defaults to 'nightly', but can be modified in case the latest nightly is broken.
   NIGHTLY_TOOLCHAIN: nightly
 
+# If a new commit is pushed to the PR, cancel any old CI checks that might still be running.
 concurrency:
-  group: ${{github.workflow}}-${{github.ref}}
-  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
What are the chances of automating the “linting” that’s happening here? Eg what’s to stop someone coming back in a month and cleaning this up in reverse?

Reply:
Not much, in all honesty. There are linters that we can add that will force a consistent style, but the CI is already doing so much.

Reply:
Yeah as far as YAML formatters that exist that I know of, it's either `prettier`, and I just found https://github.com/google/yamlfmt, which would arguably be easy and quick to install and run in CI compared to prettier.

Anyway, shame there's no amazing GH Actions tooling out there for formatting their files.

Reply:
Yeah. I use the [Github Actions Extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions) locally, but it doesn't do anything formatting-related.